### PR TITLE
mspb-215: fix teammate device payload

### PR DESCRIPTION
**Pull request by mavriciomart**
5.0 PR: https://github.com/2600hz/monster-ui-voip/pull/409
5.1 PR: https://github.com/2600hz/monster-ui-voip/pull/410

### CODE REVIEW

File: submodules/devices/devices.js [hunk 1]
```diff
@@ -1232,10 +1232,14 @@ define(function(require) {
 					smartphone: _.merge({}, sipSettings, callForwardSettings),
 					softphone: _.merge({}, sipSettings),
 					teammate: _.merge({
+						caller_id_options: {
+							outbound_privacy: "none"
+						},
+						ignore_completed_elsewhere: false,
```
Comment (harenson) [inline submodules/devices/devices.js:1238]: This one should go within a `sip` object.

Comment (harenson) [reply submodules/devices/devices.js:1238]: Here is the schema for devices requests https://github.com/2600hz/kazoo-crossbar/blob/master/priv/couchdb/schemas/devices.json#L315-L318

**Review by harenson — CHANGES_REQUESTED** [review]

File: submodules/devices/devices.js (hunk 1 above)
Comment (mavriciomart) [reply submodules/devices/devices.js]: Fixed :)

**Review by masmerino13 — APPROVED** [review]
lgtm

**Review by harenson — APPROVED** [review]
:+1:

**Pull request merged**